.github/workflows/torchbench-nightly-mps.yml

```diff
@@ -0,0 +1,48 @@
+name: torchbench-nightly-mps
+
```

Comment:
Is this supposed to run nightly? If so, I think you need to specify the time when it should run, e.g:
```
on:
  workflow_dispatch:
  schedule:
    - cron: '00 18 * * *' # run at 6:00 PM UTC
```

Reply:
That's the intention, but having it as an on-commit workflow makes it easier to debug. I'll leave this open so I remember to change it back to a nightly build before merging